job-dsl-plugin/src/main/groovy/javaposse/jobdsl/plugin/ExecuteDslScripts.java

```diff
@@ -77,6 +77,12 @@
         private String scriptText;
         private boolean ignoreMissingFiles;
 
+        // Snippet generation requires a @DataBoundConstructor.
+        // A DataBoundConstructor can take parameters whose names match
+        // databound properties.  Databound properties can be
+        // "public final" fields or "get/set" pairs. You cannot omit the getters.
+        // You cannot take constructor paramters whose names do not meet the criteria
+        // of databound properties.
```

Comment:
@jglick 
Fair point, but these requirements are apparent not self-documenting or known by all.  I figured it was worth explaining.